lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
During deployment, we'll need to ask devops to first rollback the migrations of the grades app.